migrations/20200718111257_todos.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS todos
```

Comment:
Boom!
```
sqlite> .schema
CREATE TABLE _sqlx_migrations (
    version BIGINT PRIMARY KEY,
    description TEXT NOT NULL,
    installed_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    success BOOLEAN NOT NULL,
    checksum BLOB NOT NULL,
    execution_time BIGINT NOT NULL
);
CREATE TABLE todos
(
    id          INTEGER PRIMARY KEY NOT NULL,
    description TEXT                NOT NULL,
    done        BOOLEAN             NOT NULL DEFAULT 0
);
```

Reply:
Also, I guess the sql client I normally use (Sequel Ace) doesn't support sqlite connections. the sqlite3 cli is ok but I found this as well if you want a GUI client, it seems to support pretty much everything so could be used for other projects if you want a single app for all database types. It is a trial version so not sure of the limits..https://tableplus.com

Reply:
<img width="998" alt="Screenshot 2023-10-15 at 8 56 23 AM" src="https://github.com/mblydenburgh/postie/assets/36665948/ec0db811-66ca-41ca-b3d9-bf381f896acb">

Reply:
I ended up finding a vscode plugin that works pretty well, also this web app is nice for viewing your whole schema at once: https://sqliteviewer.app/

Reply:
nice, I haven't worked with sqlite in a long time so I forgot that its heavily based around that .db file so a web app to render those details makes total sense